alibi_detect/cd/pytorch/lsdd.py

```diff
@@ -15,7 +15,8 @@ def __init__(
             preprocess_x_ref: bool = True,
             update_x_ref: Optional[Dict[str, int]] = None,
             preprocess_fn: Optional[Callable] = None,
-            sigma: Optional[np.ndarray] = None,
+            # sigma: Optional[np.ndarray] = None,
+            kernel: BaseKernel = GaussianRBF(),
```

Comment:
Perhaps confusingly, LSDD isn't a kernel method for which one can plug in their own kernel. The method is specifically tied to the GaussianRBF kernel. Therefore we should go back to not offering a `kernel` argument.

Reply:
(Obviously this applied across all the affected LSDD files. I won't duplicate the comment.)

Reply:
Fixed, now it just specifies RBF by default.